src/functions.py

```diff
@@ -1,6 +1,12 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even (num):
+  if num % 2 == 0:
```

Comment:
To simplify the function, you can also write it as:

```suggestion
  return num % 2 == 0
```

And remove the if and else 😄